cluster_experiments/dimension.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+
+
+class Dimension:
```

Comment:
is this class supposed to have more logic? if not, I'd consider making it a dataclass